tests/drivers/stepper/drv84xx/api/src/main.c

```diff
@@ -240,13 +209,12 @@ ZTEST_F(drv84xx_api, test_run_positive_direction_correct_position)
 	uint64_t step_interval = 20000000;
 	int32_t steps = 0;
 
-	(void)stepper_enable(fixture->dev);
-	(void)stepper_set_microstep_interval(fixture->dev, step_interval);
-	(void)stepper_run(fixture->dev, STEPPER_DIRECTION_POSITIVE);
+	(void)stepper_set_microstep_interval(fixture->dev, CONFIG_STEPPER_IDX, step_interval);
+	(void)stepper_run(fixture->dev, CONFIG_STEPPER_IDX, STEPPER_DIRECTION_POSITIVE);
 	k_busy_wait(110000);
 
-	(void)stepper_get_actual_position(fixture->dev, &steps);
-	zassert_true(IN_RANGE(steps, 4, 6), "Current position should be between 4 and 6 but is %d",
+	(void)stepper_get_actual_position(fixture->dev, CONFIG_STEPPER_IDX, &steps);
+	zassert_true(IN_RANGE(steps, 1, 6), "Current position should be between 4 and 6 but is %d",
```

Comment:
```suggestion
	zassert_true(IN_RANGE(steps, 1, 6), "Current position should be between 1 and 6 but is %d",
```

Missed this previously (same goes for negative direction as well).
Also, why did you go from 4 to 1?

Reply:
Just reran these tests on native sim a couple of times on my pc (including random ordering) and they all passed, even with the 4-6 range.

Concerning the accuracy, that is because these tests were originally developed for tests on real hardware.

Reply:
Okay then let's revert this change :) Probably was a mixup with QEMU.